etna/pipeline/pipeline.py

```diff
@@ -0,0 +1,59 @@
+from copy import deepcopy
+from typing import Iterable
+
+from etna.datasets import TSDataset
+from etna.models.base import Model
+from etna.transforms.base import Transform
+
+
+class Pipeline:
+    """Pipeline of transforms with a final estimator."""
+
+    def __init__(self, model: Model, transforms: Iterable[Transform] = [], horizon: int = 1):
```

Comment:
Should not use mutable objects as defaults 
```suggestion
    def __init__(self, model: Model, transforms: Iterable[Transform] = (), horizon: int = 1):
```